config.py

```diff
@@ -57,6 +68,13 @@ class Config(object):
     PARTY_AUTH = (PARTY_USERNAME, PARTY_PASSWORD)
     PARTY_RESPONDENTS_PER_PAGE = os.getenv('PARTY_RESPONDENTS_PER_PAGE', 25)
 
+    RAS_NOTIFY_SERVICE_URL = os.getenv('RAS_NOTIFY_SERVICE_URL', 'http://notify-gateway-service/emails/')
```

Comment:
I would be more inclined to remove the RAS from these variables as we have been moving away from using RAS or RM as a prefix to the variables.

Reply:
Is our preference towards consistency or moving away from RAS/RM? This is consistent with other services

Reply:
Usually I'd say consistency trumps all, but changing these from ras_<thing> to <thing> is a pain so we should make that job easier for us when the time comes by having 1 less thing to do.